YoCode/LoadingAnimation.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+
+namespace YoCode
+{
+    public class LoadingAnimation
+    {
+        static string dots;
+        public static void RunLoading()
+        {
+            var cursorPos = Console.CursorTop;
+            Console.WriteLine(messages.Fireplace);
+            Console.WriteLine(messages.ParagraphDivider);
+            Console.WriteLine($"Get comfortable, YoCode is about to finish gathering{Environment.NewLine}your results");
+            Console.WriteLine(messages.ParagraphDivider);
+
+            while (true)
+            {
+                ClearLine();
+                Console.Write(String.Format("-----------------------[Loading{0,-3}]----------------------", dots));
```

Comment:
I don't think the 3 dots in the middle of a line are obvious enough. Maybe leave off the `------` stuff so that `Loading...` appears on its own line?